JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -23,6 +26,16 @@ Manometer,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device used to measu
 Accelerometer,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A tool that measures proper acceleration.,https://www.theworldavatar.com/kg/ontodevice/,Accelerometer
 ElectricityMeter,Class,https://saref.etsi.org/core/Meter,IS-A,,,,"A device that measures the amount of electric energy consumed by a residence, a business, or an electrically powered device.",https://www.theworldavatar.com/kg/ontodevice/,Electricity meter
 ThermalEnergyMeter,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device that is able to measure thermal energy and is usually used to measure energy from hydronic-based heating and cooling systems.,https://www.theworldavatar.com/kg/ontodevice/,Thermal energy meter
+Grindometer,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device that measures the particle size of ground materials.,https://www.theworldavatar.com/kg/ontodevice/,Grindometer
+Calorimeter,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device that measures the heat of chemical reactions or physical changes.,https://www.theworldavatar.com/kg/ontodevice/,Calorimeter
+DifferentialScanningCalorimeter,Class,Calorimeter,IS-A,,,,A differential scanning calorimeter used for analyzing thermal properties.,https://www.theworldavatar.com/kg/ontodevice/,Differential scanning calorimeter
+Coulometer,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device used to measure the amount of electricity passed through an electrolyte.,https://www.theworldavatar.com/kg/ontodevice/,Coulometer
+pHMeter,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device that measures the acidity or alkalinity of a solution.,https://www.theworldavatar.com/kg/ontodevice/,pH meter
+Spectrometer,Class,https://saref.etsi.org/core/Meter,IS-A,,,,A device used to measure and analyze the spectrum of light.,https://www.theworldavatar.com/kg/ontodevice/,Spectrometer
+InfraredSpectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that analyzes infrared light spectra.,https://www.theworldavatar.com/kg/ontodevice/,Infrared spectrometer
+MassSpectrometer,Class,Spectrometer,IS-A,,,,A device used to measure the mass-to-charge ratio of ions.,https://www.theworldavatar.com/kg/ontodevice/,Mass spectrometer
+UltravioletVisibleSpectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that measures ultraviolet and visible light.,https://www.theworldavatar.com/kg/ontodevice/,Ultraviolet-visible spectrometer
+Xray_Spectrometer,Class,Spectrometer,IS-A,,,,A spectrometer that analyzes X-ray spectra.,https://www.theworldavatar.com/kg/ontodevice/,X-ray spectrometer
```

Comment:
The naming seems inconsistent. None of the other class names contain underscores. For consistency, I'd recommend removing the underscore.